devops/dev/values.yaml

```diff
@@ -101,6 +101,9 @@ did-auth-proxy-helm:
     REDIS_HOST: did-auth-proxy-redis-master.did-auth-proxy.svc.cluster.local
     REDIS_PORT: 6379
     REDIS_PASSWORD: redis
+    IPFS_HOST: ipfs.infura.io
+    IPFS_PORT: 5001
+    IPFS_PROJECTID: 2DXJGR6JFXIE612O8bUvjXGIBIP
```

Comment:
Nice. Do you great an additional project beyond the ones we have listed in the confluence page?

Reply:
Updated